batch/snippets/src/test/java/com/example/batch/CreateResourcesIT.java

```diff
@@ -238,18 +237,8 @@ public void createBatchCustomEventTest()
 
     Assert.assertTrue(job.getTaskGroupsList().stream()
             .anyMatch(event -> event.getTaskSpec().getRunnablesList()
-                    .stream().anyMatch(runnable
-                            -> runnable.getDisplayName().contains(displayName1))));
-
-    Assert.assertTrue(job.getTaskGroupsList().stream()
-            .anyMatch(event -> event.getTaskSpec().getRunnablesList()
-                    .stream().anyMatch(runnable
-                            -> runnable.getDisplayName().contains(displayName2))));
-
-    Assert.assertTrue(job.getTaskGroupsList().stream()
-            .anyMatch(event -> event.getTaskSpec().getRunnablesList()
-                    .stream().anyMatch(runnable
-                            -> runnable.getDisplayName().contains(displayName3))));
+                    .stream().allMatch(runnable -> runnable.getDisplayName()
+                            .matches(displayName1 + "|" + displayName2))));
```

Comment:
This will match either of the display names for each runnable.
We want to only verify if both names are present in the runnable list. Modify to code to something like:

```
Stream.of(displayName1, displayName2)
    .allMatch(displayName -> job.getTaskGroupsList().stream()
        .flatMap(event -> event.getTaskSpec().getRunnablesList().stream())
        .anyMatch(runnable -> runnable.getDisplayName().equals(displayName))
    );
```